static/js/play.js

```diff
@@ -176,16 +178,16 @@ async function finish() {
 function hideElements() {
     
     var hide = ["reference","mw-editsection","reflist","portal","refbegin", "sidebar", "authority-control", "external", "sistersitebox"]
-    for(i=0; i<hide.length; i++) {
+    for(var i=0; i<hide.length; i++) {
```

Comment:
small thing, but if you're going through and fixing these, you should use `let`/`const` instead of `var`

Reply:
done, made those changes at first because I had to change the js type to module